src/api/models/nftsModel.ts

```diff
@@ -0,0 +1,36 @@
+import { DATABASE } from "../config/database";
+
+export const nftsModel = {
+	fetchEventStream: async function (tokenId) {
+		const connection = await DATABASE.getConnection();
+		let res = [{}];
+		try {
+			res = await connection.execute(
+				`SELECT * FROM event_tracker where stream_id = ? ORDER BY version ASC`,
```

Comment:
worth to consider using an ORM like https://www.prisma.io/docs/ to help you interface with DB, safer

Reply:
will check
=

Reply:
done